oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,213 @@
+in the `Open edX Slack <https://openedx.slack.com/>`_.
+
+.. _selecting new members:
+
+Adding New Core Contributors
+----------------------------
+
+New Core Contributors are determined via a nomination process - CCs may nominate
+new members, or existing community members may put themselves forward for a role.
+
+Current project CCs should always be on the lookout for contributors who demonstrate the
+"3 Cs" (Commitment, Conduct, and Caliber), follow the `Code of Conduct
+<https://open.edx.org/code-of-conduct/>`_, are respectful of other community members, are polite and
+welcoming to all in the community, and have been an overall active community member for some time.
+When a person like this comes to the attention of the CC, the CC should reach out and ask if
+that person would like to be nominated as a CC.
+
+Community members who feel they fit these criteria listed above should feel empowered to
+nominate themselves, as well.
+
+Note that when a new Core Contributor role is being defined and piloted, however,
+nominations should proceed only once the Program Administrator is confident that new Core
+Contributors in that role can be supported. Reach out to the Administrator when there is
+any uncertainty around the status of the role; Administrators can be contacted at ``cc-program-admins@tcril.org``.
+
+New Core Contributor Nomination Process
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The process for nomination is as follows:
+
+#. An existing CC ("sponsor") choses to nominate a community member ("sponsored candidate") for a new CC
+   position, OR, a current community member ("organic candidate") decides to put themselves forth for nomination.
+
+#. The sponsor or organic candidate posts a new public nomination thread on Discourse, in the `Core Contributor
+   discussion space <https://discuss.openedx.org/c/working-groups/core-contributors/36>`_.
+
+   #. This post introduces the candidate, describes in a few paragraphs why they are suited to join the program,
+      and provides links to examples of previous Open edX work (for example, pull requests).
+   
+   #. The post should also indicate the candidate's expected commitment to contribution - if the candidate is
+      part of an organization, this should be covered as part of the organization's signature to the Declaration
+      of Commitment to the Core Contributor Program, as covered in the `program administration`_ section.
+   
+   #. Finally, the post must mention the date when the comment period ends, which is one month from the start of
+      the thread.
+
+#. The sponsor or organic candidate posts a link to the Discourse post in the ``#core-contributors`` Slack room,
+   as well as any other Slack rooms relevant to the CC role being discussed (for example, repo-specific rooms for
+   coding contributors, translation team rooms, etc). **The messages must indicate the dates of the comment period.**
```

Comment:
Is the process the same for a contributor who wants to change or add roles?

Reply:
I think that makes sense. What is your opinion?

Reply:
It seems like a lot of overhead for some lateral moves, but I can see an argument for defaulting to everything being consistent.

Reply:
Yeah - the more I think about it the more I want it to be consistent. Sure, it may be quick to go from coder to translator, but going from forum mod to coder would want the same amount of input. I could maybe see shortening the comment period one or two weeks for people changing/adding roles, pending judgement of the person nominating.

Relatedly, is a month too long of a comment period? I threw it out as a strawman - I don't think it should be any longer than that. But I also didn't want someone nominating someone on 21 Dec and accepting them on 4 Jan - no one would have time to comment.

Reply:
Whoop, I'm reading from top to bottom and made a related comment above.  IMHO, yeah, sometimes a month period would feel really long.  Maybe like... two/three weeks of business days or something?  That way it's a bit resilient to holiday breaks?  Just riffing here.

Reply:
I have come to terms that saying "2 weeks or more, being mindful of holiday periods"

Reply:
The duration seem fine to me - we can start with a duration and adapt as needed anyway. One month wasn't shocking to me though - getting a core contributor role isn't something that needs to be light or fast. This is a senior responsibility, that comes to reward months or years of consistent contribution to the platform and the community. Having a comment period that ensures a maximum of people can review it would help ensuring the quality of the core contributors being nominated.